src/vimhelp/html.clj

```diff
@@ -2,20 +2,27 @@
   (:require
    [clojure.java.io :as io]
    [clojure.string :as str]
-   [hiccup.core :as hiccup]
-   [hiccup.page :as page])
+   [hiccup2.core :as hiccup])
   (:import
    java.net.URLEncoder))
 
 (defn- url-encode
   [s]
   (URLEncoder/encode s "UTF-8"))
 
-(defn- replace-spaces
+(defn replace-spaces
   [s]
-  (-> s
-      (str/replace " " "&nbsp;")
-      (str/replace "\t" "&nbsp;&nbsp;&nbsp;&nbsp;")))
+  (->> s
+       (partition-by #(case % \space 0 \tab 0 1))
+       (map (fn [char-seq]
+              (if (#{\space \tab} (first char-seq))
```

Comment:
It may be good to define a private/local var for `#{\space \tab}`

e.g. `(let [spaces #{\space \tab}] ...)`